src/features/payment/PaymentScreen.tsx

```diff
@@ -679,13 +682,15 @@ const PaymentScreen = () => {
                   >
                     {t('payment.send')}
                   </Text>
-                  <TouchableOpacityBox
-                    onPress={navigation.goBack}
-                    padding="l"
-                    hitSlop={hitSlop}
-                  >
-                    <Close color={primaryText} height={16} width={16} />
-                  </TouchableOpacityBox>
+                  <Box hitSlop={hitSlop} padding="s">
```

Comment:
Why doesnt the `TouchableOpacityBox` work for these?

Reply:
It does work I am just reusing `IconPressedContainer` as it adds the haptic feedback per Petes feedback